templates/systemd/elasticsearch.j2

```diff
@@ -3,6 +3,8 @@ Description=Elasticsearch-{{es_instance_name}}
 Documentation=http://www.elastic.co
 Wants=network-online.target
 After=network-online.target
+{# Directive 'WorkingDirectory' creates an implicit dependecy and can be omitted #}
```

Comment:
Did you mean to leave this in? This comment mentions the `WorkingDirectory` directive but you are adding in `RequiresMountsFor`.

Reply:
The `WorkingDirectory` directive is used in [line 18](https://github.com/elastic/ansible-elasticsearch/blob/e999988560469d69d353ffb472c7f83ab685dccd/templates/systemd/elasticsearch.j2#L18).
It **implicitly** creates the necessary `After=`/`Requires=` directives for `{{es_home}}`, therefore we can skip adding `{{es_home}}` to the list of folders in `RequiresMountsFor`.

I'll refine the comment to make it more clear, why I skipped `{{es_home}}`.

Do you prefer Jinja2 comments here, i.e. comments that are no longer in the templated file, or "regular" comments?

Reply:
Updated the Jinja comment for now - since we're using template variables, it would imho be confusing in the final systemd-unitfile.

Reply:
Thank you for clarifying and updating it!